release-please-config.json

```diff
@@ -11,12 +11,10 @@
     "packages/interfaces": {},
     "packages/message-hash": {},
     "packages/enr": {},
-    "packages/peer-exchange": {},
     "packages/core": {},
-    "packages/dns-discovery": {},
     "packages/message-encryption": {},
     "packages/sdk": {},
     "packages/relay": {},
-    "packages/local-peer-cache-discovery": {}
```

Comment:
should be removed from here as well - https://github.com/waku-org/js-waku/blob/fb41f4c723238628f398dff4bc423edee8fd1c6e/.release-please-manifest.json#L7